include/battle_scripts.h

```diff
@@ -86,6 +86,7 @@ extern const u8 BattleScript_WrapFree[];
 extern const u8 BattleScript_LeechSeedFree[];
 extern const u8 BattleScript_SpikesFree[];
 extern const u8 BattleScript_MonTookFutureAttack[];
+extern const u8 BattleScript_FutureAttackFail[];
```

Comment:
I'm surprised this compiles because the string doesn't exist anymore, haha

Reply:
fixed